### PR TITLE
PHP8 support

### PR DESCRIPTION
**Pull request by RobjS**
This PR:

- Adds PHP 8.1 and up support (and upgrades to PHPUnit 9 to facilitate this)
- Implements the standard dxw PHP CS Fixer config

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
         }
     },
     "require": {
+        "php": "^7.4||^8.1",
```
Comment (snim2) [inline composer.json:17]: Does 8.2 work OK?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [Earlier Releases]
+
+Releases up to and including v1.0.0 predate this changelog.
```
Comment (snim2) [inline CHANGELOG.md]: This is not a blocking change, but do we have anything that describes the features here that we could point to?

**Review by snim2 — APPROVED** [review]
lgtm

File: composer.json (hunk 1 above)
Comment (RobjS) [reply composer.json:17]: Yes, CI is passing in 8.1 & 8.2.

File: CHANGELOG.md (hunk 2 above)
Comment (RobjS) [reply CHANGELOG.md]: I've added a link to the README.

**Pull request merged**